src/frontend/gatsby/src/components/SearchPage/SearchResults.jsx

```diff
@@ -69,7 +76,11 @@ const SearchResults = ({ text, filter }) => {
       {data && data.length ? (
         <Timeline pages={data} nextPage={() => setSize(size + 1)} />
       ) : (
-        <h1>No search results</h1>
+        <div className={classes.noResults}>
+          <img src={NotFound} alt="Not Found" height={200} width={200} />
+          <h1> We have not found anything</h1>
+          <h1> Try searching for something else</h1>
```

Comment:
You can't have more than one `<h1>` in a page.  Use CSS styling to achieve the look/sizing you want instead of relying on `<h1>` twice.  Also, maybe combine these into a single message so the `<h1>` works?

Reply:
Got it